pykechain/enums.py

```diff
@@ -69,3 +69,10 @@ class ComponentXType(Enum):
     SUPERGRID = "superGrid"
     PAGINATEDSUPERGRID = "paginatedSuperGrid"
     FILTEREDGRID = "filteredGrid"
+
+
+class Status(Enum):
```

Comment:
lets call this `ActivityStatus` as it bound to actvity. Don;t worry as we will get more statussi.

We might also want to check the 'allowed' transitions in the activity json to ensure that we do not set a status that is invalid to begin with.